saltproc/app.py

```diff
@@ -21,6 +21,139 @@
 import numpy as np
 
 
+def run():
```

Comment:
Did anything about this function change besides its location?

Reply:
Other than using the new function names and signatures, (and some adjustments to the print statements), no.